src/Agent.Sdk/SecretMasking/LoggedSecretMasker.cs

```diff
@@ -117,7 +124,6 @@ public void AddValueEncoder(ValueEncoder encoder)
         public void AddValueEncoder(ValueEncoder encoder, string origin)
         {
             this.Trace($"Setting up value for origin: {origin}");
-            this.Trace($"Length: {encoder.ToString().Length}.");
```

Comment:
>ToString [](http://example.com/codeflow?start=42&length=8)

I'm not sure why the length of this value was important to log. Was encoder.ToString() intended to log the value encoder type name? Perhaps we should log encoder.GetType().Name?

Reply:
`encoder.GetType().Name` is going to be `"ValueEncoder"` :) So is `encoder.ToString()` so this would log 'Length: 12' always before. 😊

`encoder.Method` could be useful but isn't reliable: it will be junk if an anonymous method is ever used. For the current uses, this would only repeat information that is also passed in the origin.

I think origin is sufficient.

Reply:
Oh, I see, 'origin' is just a bad name. :) This isn't the origin of the call, it's a named identifier for the value encoded. This is confusion I'd suggest cleaning up.

I can appreciate that for the general logging scenario, we have an arbitrary string that gets passed around that provides additional data on the log call. For `AddValueEncoder`, however, what we're looking for is the unique name of the encoder, so that we know it was added.